.travis.yml

```diff
@@ -19,10 +19,10 @@ install:
 before_script:
 - go version
 - docker version
-- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.15.12/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.17.17/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
Why 1.17 when it already reached end of life? Why not something more up to date?

Reply:
We want to get to latest. But... we originally attempted an update from 1.12 to 1.22 and that didn't go so well. 😅 
We'll try to keep the updates coming.

Reply:
Make sense, thanks!